docs/manual/advanced-features.tex

```diff
@@ -915,6 +915,9 @@
 \item
   \ahrefloc{index-checker}{Index Checker} for array accesses
   (see \chapterpageref{index-checker})
+\item
```

Comment:
The lists in `introduction.tex` and `advanced-features.tex` do not include the Must Call Checker or the Called Methods Checker.

Reply:
Good point! https://github.com/typetools/checker-framework/pull/4716